_layouts/simple.html

```diff
@@ -53,29 +53,27 @@ <h2 class="addresses-header">Our locations</h2>
     {% endif %}
     {% if page.title == "Privacy Policy" %}
     <div class="story-list">
-        <span><a href="#English">English</a> /</span>
-        <span><a href="#Swahili">Swahili</a> /</span> </br>
-        <span><a href="#French">French</a> /</span>
-        <!-- <span><a href="#Hindi">Hindi</a> /</span> -->
+      <span><a href="/privacy-policy-english/" {% if page.url == "/privacy-policy-english/" %}style="color: #1D214F;"{% endif %}>English /</a></span>
+      <span><a href="/privacy-policy-swahili/" {% if page.url == "/privacy-policy-swahili/" %}style="color: #1D214F;"{% endif %}>Swahili /</a></span> <br>
+      <span><a href="/privacy-policy-french/" {% if page.url == "/privacy-policy-french/" %}style="color: #1D214F;"{% endif %}>French /</a></span>
     </div>
     {% endif %}
     {% if page.title != "Contact" %}
     <div class="success-story-wrapper">
-      {% if page.title == "Privacy Policy" %}<h2 id="English">English</h2>{% endif %}
+      {% if page.title == "Privacy Policy" %}
+      <!-- <h2 id="English">English</h2> -->
+      {% endif %}
```

Comment:
Ok

Reply:
> @nashthecoder while testing this locally I'm noticing that the top navbar changes from the english page to the other 2 pages. English has `Home What we do About us Join us Impact` and the others have `Home What we do Impact Join us About us`.
> 
> Also noticed that the english version does not show the title of the policy like the french and swahili do.
> 
> Can you check if there is a need to reuse the `_simple` layout here or if we could use a specific layout for the privacy-policy pages? To avoid all the conditionals on that layout.

On it.